### DISCUSSION TITLE
大佬们 如果我有一个react项目如何打包成离线桌面应用呢

### DISCUSSION THREAD

**Discussion by Deja-vuuu** [tw93/Pake#829] [Q&A]
首先我的项目不想部署在服务器上，就想直接把build出来的react文件 再打包成桌面应用。
我试了一下好像不太行

<img width="876" alt="image" src="https://github.com/user-attachments/assets/31e26073-cacc-4632-b933-cb1d751d07c3">

**Comment by Sjj1024** [comment]
> 首先我的项目不想部署在服务器上，就想直接把build出来的react文件 再打包成桌面应用。
> 
> 我试了一下好像不太行
> 
> 
> 
> <img width="876" alt="image" src="https://github.com/user-attachments/assets/31e26073-cacc-4632-b933-cb1d751d07c3">
> 
> 

我的PakePlus项目可以的
